core/src/SYCL/Kokkos_SYCL_Parallel_Reduce.hpp

```diff
@@ -271,8 +271,8 @@ class ParallelReduce<CombinedFunctorReducerType, Kokkos::RangePolicy<Traits...>,
           instance.scratch_flags(sizeof(unsigned int)));
 
       auto reduction_lambda_factory =
-          [&](sycl::local_accessor<value_type, 1> local_mem,
-              sycl::local_accessor<unsigned int, 1> num_teams_done,
+          [&](sycl::local_accessor<value_type> local_mem,
+              sycl::local_accessor<unsigned int> num_teams_done,
```

Comment:
To other reviewers , `sycl::local_accessor<class DataT, int Dimensions = 1, ...>`